src/ratingservice/main.py

```diff
@@ -68,6 +68,7 @@ def makeError(code, message):
 
 
 def makeResult(data):
+    print("2", data)
```

Comment:
Looks like a debugging string was accidentally committed

Reply:
OMG. will be removed